### PR TITLE
Select dropdown bug fix

### PR DESCRIPTION
**Pull request by abericExecom**

### CODE REVIEW

**Review by MiljkovicVeljko — APPROVED** [review]

**Review by mkovacevicExecom — APPROVED** [review]

File: client-app/src/app/shared/components/select-dropdown/select-dropdown.component.html [hunk 1]
```diff
@@ -1,40 +1,39 @@
-<div class="selector" appClickOutside (clickOutside)="closeDropdown()">
-  <div class="d-flex align-items-center">
+<div class="selector position-relative" appClickOutside (clickOutside)="closeDropdown()">
+  <div
+    [class.item-selector-location-item]="isLocationSelector"
+    class="item-selector-item d-flex align-items-center px-4"
```
Comment (aberonja) [inline client-app/src/app/shared/components/select-dropdown/select-dropdown.component.html:4]: Reorder.

File: client-app/src/app/shared/components/select-dropdown/select-dropdown.component.html [hunk 2]
```diff
@@ -1,40 +1,39 @@
...
+    (click)="showDropdownItemSelection()"
+  >
+    <span [class.item-selector-location-text]="isLocationSelector" class="mx-2 item-selector-text">{{
```
Comment (aberonja) [inline client-app/src/app/shared/components/select-dropdown/select-dropdown.component.html:7]: Reorder [class] and classes.

File: client-app/src/app/shared/components/select-dropdown/select-dropdown.component.html [hunk 3]
```diff
@@ -1,40 +1,39 @@
...
+      class="item-selector-img ml-auto"
+      alt="caret down"
+      src="assets/images/{{ selectorImg }}.svg"
+      [class.item-selector-location-img]="isLocationSelector"
+    />
+  </div>
+  <div
+    *ngIf="showItemDropdown"
+    [class.select-dropdown-container-location]="isLocationSelector"
+    class="position-absolute select-dropdown-container"
```
Comment (aberonja) [inline client-app/src/app/shared/components/select-dropdown/select-dropdown.component.html:20]: Reorder.

File: client-app/src/app/shared/components/select-dropdown/select-dropdown.component.html [hunk 4]
```diff
@@ -1,40 +1,39 @@
...
       <div
-        class="item-selector-item d-flex align-items-center px-4"
-        [class.item-selector-location-item]="isLocationSelector"
+        class="mx-2 d-flex flex-fill align-items-center"
+        [class.border-top]="!isLocationSelector"
+        (click)="selectItem(item)"
       >
-        <span [class.item-selector-location-text]="isLocationSelector" class="mx-2 item-selector-text">{{
-          currentItem.text | translate
+        <span [class.item-selector-location-text]="isLocationSelector" class="mr-auto item-selector-text pt-4">{{
```
Comment (aberonja) [inline client-app/src/app/shared/components/select-dropdown/select-dropdown.component.html:34]: Reorder classes.

**Review by aberonja — CHANGES_REQUESTED** [review]

**Review by aberonja — APPROVED** [review]

**Pull request merged**